src/Utilities/_DataValidation.php

```diff
@@ -148,6 +148,14 @@ function validateElements(array $elements): void
     }
 }
 
+if (!function_exists('validateItems')) {
+    function validateItems(array $items): void
+    {
+        $keys = array_keys($items);
+        validateKeys($keys);
```

Comment:
Kind of picky, but rather than adding a new function here can you just move this logic to the `setBatch()` call in the data client, replacing `validateItems($items)` with `validateKeys(array_keys($items))`? Because this method doesn't actually do any validation of the items in the array, there's really no compelling reason for the indirection.